src/modules/constants.ts

```diff
@@ -160,6 +161,51 @@ const POLYGON_NETWORK: NetworkInfo = {
   gnosisTxService: 'https://safe-transaction.polygon.gnosis.io/',
 };
 
+const RINKEBY_TEST_NETWORK: NetworkInfo = {
+  name: 'Rinkeby Testnet',
+  chainId: 4,
+  shortName: 'RIN',
+  contractAddressLink: '',
+  gnosisTxService: 'https://safe-transaction.rinkeby.gnosis.io/',
+};
+
+/*
+ * @NOTE: Although officially supported, 
+ * I cannot find the Safe Transaction API address for those commented out below.
+
+const KOVAN_TEST_NETWORK: NetworkInfo = {
+  name: 'Kovan Testnet',
+  chainId: 42,
+  shortName: 'KOV',
+  contractAddressLink: '',
+  gnosisTxService: '',
+};
+
+const ROPSTEN_TEST_NETWORK: NetworkInfo = {
+  name: 'Ropsten Testnet',
+  chainId: 3,
+  shortName: 'ROP',
+  contractAddressLink: '',
+  gnosisTxService: '',
+};
+
+const ENERGY_WEB_CHAIN_NETWORK: NetworkInfo = {
+  name: 'Energy Web Chain',
+  chainId: 246,
+  shortName: 'EWT',
+  contractAddressLink: '',
+  gnosisTxService: '',
+};
+*/
+
+const VOLTA_TEST_NETWORK: NetworkInfo = {
+  name: 'Energy Web Volta Testnet',
+  chainId: 73799,
+  shortName: 'VT',
+  contractAddressLink: '',
+  gnosisTxService: 'https://safe-transaction.volta.gnosis.io/',
+};
+
```

Comment:
These are test networks, we don't need to include them as options. I think just Rinkeby is fine to include.